File: langchain.js-Demo/src/models/ollama.ts
import { ChatOllama } from "@langchain/ollama";

import { env } from "../config/env.js";

export function createOllamaModel() {
  return new ChatOllama({
    baseUrl: env.OLLAMA_BASE_URL,
    model: env.OLLAMA_MODEL,
  });
}

File: langchain.py学习/src/models/ollama.py
from langchain_ollama import ChatOllama

from src.config.env import load_env


def create_ollama_chat_model() -> ChatOllama:
    env = load_env()
    return ChatOllama(
        base_url=env.ollama_base_url,
        model=env.ollama_model,
    )
